timeseries/src/autogluon/timeseries/models/sktime/abstract_sktime.py

```diff
@@ -67,21 +67,27 @@ def __init__(
             hyperparameters=hyperparameters,
             **kwargs,
         )
-        self.skt_forecaster: Optional[BaseForecaster] = None
+        self.sktime_forecaster: Optional[BaseForecaster] = None
         self._fit_index: Optional[pd.Index] = None
 
-    def _get_skt_forecaster(self, sp: Optional[int] = None) -> BaseForecaster:
-        """Return the sktime forecaster object for the model"""
-        params = self._get_model_params().copy()
-        if sp is not None and sp > 1:
-            params["sp"] = sp
-            params["seasonal"] = "add"
+    def _get_sktime_forecaster_init_args(self, min_length: int, inferred_period: int = 1):
+        """Get arguments that will be passed to the sktime forecaster at initialization."""
+        return self._get_model_params().copy()
+
```

Comment:
We now call `self._get_model_params().copy()` inside `_get_sktime_forecaster_init_args`, so no more in-place modifications.

Reply:
Just nit that this results in a shallow copy, but should be good enough for us 💯